src/013-static-connection-pool.md

```diff
@@ -0,0 +1,39 @@
+# How to make a static Database Connection
+
+> Read first: [Documentation on Connection Pool](https://www.sea-ql.org/SeaORM/docs/install-and-config/connection/)
+
+Everytime we access the database, we need to establish a connection, or as we call it, a "Connection Pool" beforehand. The most trivial way is to create the connection everytime we need to access it:
+
+```rust, no_run
+// Establish connection
+let db = Database::connect(url).await?;
+
+/* Some operations on db */
+
+// Close connection
+db.close().await?;
+```
+
+This connection is, however, only locally accessible, and each establishment of connection is very costly. To solve it, one would like to create a static connection that only has to be established once. Doing so requires the [`lazy_static`](https://crates.io/crates/lazy_static) and [`async-std`](https://crates.io/crates/async-std) crate.
```

Comment:
Actually, one can choose any async runtime. All we want is a way to execute a future and yield the `DatabaseConnection` at the end.

I choose `async-std` as a demo in the GitHub Discussion thread.

So, one can choose any async runtime they deem appropriate. We may well say it out loud here.